include/inout.h

```diff
@@ -114,4 +114,19 @@ static inline uint8_t IO_Read(io_port_t port)
 	return IO_ReadB(port);
 }
 
+// Hardware I/O port numbers
+
+// Intel 8042 keyboard/mouse port microcontroller
+constexpr io_port_t port_num_i8042_data    = 0x60u;
+constexpr io_port_t port_num_i8042_status  = 0x64u; // read-only
+constexpr io_port_t port_num_i8042_command = 0x64u; // write-only
+
+// Intel 8255 microcontrollers
+constexpr io_port_t port_num_i8255_1 = 0x61u;
+constexpr io_port_t port_num_i8255_2 = 0x62u;
+
+// VMware communication interface
+constexpr io_port_t port_num_vmware    = 0x5658u;
+constexpr io_port_t port_num_vmware_hb = 0x5659u; // high bandwidth
+
```

Comment:
Nice!

Reply:
Oh yeah, it's a lot nicer! 🚀

Reply:
Picky: do we need `_num_` in the names? E.g. would `port_i8042_data` be sufficient?

Reply:
I think the num differentiates it from the port function call, for example:

```c++
IO_RegisterReadHandler(
    port_num_vmware,
    port_read_vmware,
    io_width_t::dword);
}
```